### PR TITLE
PLAT-124527: Fix text to be aligned in center if no icon in FanSpeedControl

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Issue: Text is not aligned in center if no icon in FanSpeedControl in sampler.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Solution: Added `noIcon` class to center text when there is no icon.

### Links
[//]: # (Related issues, references)
PLAT-124527

### Comments

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]

File: FanSpeedControl/FanSpeedControl.js [hunk 1]
```diff
@@ -98,7 +98,8 @@ const FanSpeedControlBase = kind({
 	},
 
 	computed: {
-		children: ({min, max}) => range(min, max + 1)
+		children: ({min, max}) => range(min, max + 1),
+		className: ({icon, styler}) => styler.append(!icon ? 'noIcon' : '')
```
Comment (ybsung) [inline FanSpeedControl/FanSpeedControl.js:102]:
```suggestion
		className: ({icon, styler}) => styler.append({noIcon: !icon})
```

**Comment by ybsung** [comment]
@stanca-pop-lgp 
Please update CHANGELOG.md.

**Review by ybsung — APPROVED** [review]
LGTM

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**